CleanWeather/Extensions/DoubleExtension.swift

```diff
@@ -15,6 +15,6 @@ extension Double {
     }
     
     var percentString: String {
-        return "\(Int(self.rounded()))%"
+        return "\(Int(self * 100))%"
```

Comment:
return "\((self * 100).rounded())%"

wouldnt it be better?

Reply:
Why would that be better?
I would like to have 43% displayed for example - not 40%.